From: Stefano Rivera <stefanor@debian.org>
Date: Tue, 5 Mar 2024 21:11:00 -0400
Subject: Rename manpages pip -> pip3

While we call the binary pip3, the manpages should match.

Forwarded: not-needed
---
 docs/html/conf.py               |  4 ++--
 docs/man/commands/cache.rst     |  6 +++---
 docs/man/commands/check.rst     |  6 +++---
 docs/man/commands/config.rst    |  6 +++---
 docs/man/commands/debug.rst     |  2 +-
 docs/man/commands/download.rst  |  6 +++---
 docs/man/commands/freeze.rst    |  6 +++---
 docs/man/commands/hash.rst      |  6 +++---
 docs/man/commands/help.rst      |  6 +++---
 docs/man/commands/install.rst   |  6 +++---
 docs/man/commands/list.rst      |  6 +++---
 docs/man/commands/lock.rst      |  6 +++---
 docs/man/commands/search.rst    |  6 +++---
 docs/man/commands/show.rst      |  6 +++---
 docs/man/commands/uninstall.rst |  6 +++---
 docs/man/commands/wheel.rst     |  6 +++---
 docs/man/index.rst              | 28 ++++++++++++++--------------
 17 files changed, 59 insertions(+), 59 deletions(-)

diff --git a/docs/html/conf.py b/docs/html/conf.py
index 03df87e..d736e29 100644
--- a/docs/html/conf.py
+++ b/docs/html/conf.py
@@ -122,11 +122,11 @@ def determine_man_pages() -> List[Tuple[str, str, str, str, int]]:
         )
 
     retval = [
-        ("index", "pip", "package manager for Python packages", "pip developers", 1),
+        ("index", "pip3", "package manager for Python packages", "pip developers", 1),
     ]
     for fname in raw_subcommands:
         fname_base = to_document_name(fname, man_dir)
-        outname = "pip-" + fname_base.split("/")[1]
+        outname = "pip3-" + fname_base.split("/")[1]
         description = "description of {} command".format(outname.replace("-", " "))
 
         retval.append((fname_base, outname, description, "pip developers", 1))
diff --git a/docs/man/commands/cache.rst b/docs/man/commands/cache.rst
index 8f8e197..45bac6c 100644
--- a/docs/man/commands/cache.rst
+++ b/docs/man/commands/cache.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-=========
-pip-cache
-=========
+==========
+pip3-cache
+==========
 
 Description
 ***********
diff --git a/docs/man/commands/check.rst b/docs/man/commands/check.rst
index a799742..fe20019 100644
--- a/docs/man/commands/check.rst
+++ b/docs/man/commands/check.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-=========
-pip-check
-=========
+==========
+pip3-check
+==========
 
 Description
 ***********
diff --git a/docs/man/commands/config.rst b/docs/man/commands/config.rst
index bdbeaf6..74b6f6b 100644
--- a/docs/man/commands/config.rst
+++ b/docs/man/commands/config.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-==========
-pip-config
-==========
+===========
+pip3-config
+===========
 
 Description
 ***********
diff --git a/docs/man/commands/debug.rst b/docs/man/commands/debug.rst
index b27ef2b..9f19e79 100644
--- a/docs/man/commands/debug.rst
+++ b/docs/man/commands/debug.rst
@@ -1,7 +1,7 @@
 :orphan:
 
 ==========
-pip-debug
+pip3-debug
 ==========
 
 Description
diff --git a/docs/man/commands/download.rst b/docs/man/commands/download.rst
index 5ffff66..0e2caa9 100644
--- a/docs/man/commands/download.rst
+++ b/docs/man/commands/download.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-============
-pip-download
-============
+=============
+pip3-download
+=============
 
 Description
 ***********
diff --git a/docs/man/commands/freeze.rst b/docs/man/commands/freeze.rst
index f61d87e..7dfdf64 100644
--- a/docs/man/commands/freeze.rst
+++ b/docs/man/commands/freeze.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-==========
-pip-freeze
-==========
+===========
+pip3-freeze
+===========
 
 Description
 ***********
diff --git a/docs/man/commands/hash.rst b/docs/man/commands/hash.rst
index 83d6cfc..ad28237 100644
--- a/docs/man/commands/hash.rst
+++ b/docs/man/commands/hash.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-========
-pip-hash
-========
+=========
+pip3-hash
+=========
 
 Description
 ***********
diff --git a/docs/man/commands/help.rst b/docs/man/commands/help.rst
index 19a8aa8..50b9bf2 100644
--- a/docs/man/commands/help.rst
+++ b/docs/man/commands/help.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-========
-pip-help
-========
+=========
+pip3-help
+=========
 
 Description
 ***********
diff --git a/docs/man/commands/install.rst b/docs/man/commands/install.rst
index c0e9ac6..47a48d8 100644
--- a/docs/man/commands/install.rst
+++ b/docs/man/commands/install.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-===========
-pip-install
-===========
+============
+pip3-install
+============
 
 Description
 ***********
diff --git a/docs/man/commands/list.rst b/docs/man/commands/list.rst
index de6b7d5..9843acb 100644
--- a/docs/man/commands/list.rst
+++ b/docs/man/commands/list.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-========
-pip-list
-========
+=========
+pip3-list
+=========
 
 Description
 ***********
diff --git a/docs/man/commands/lock.rst b/docs/man/commands/lock.rst
index ca1de2c..76f525f 100644
--- a/docs/man/commands/lock.rst
+++ b/docs/man/commands/lock.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-========
-pip-lock
-========
+=========
+pip3-lock
+=========
 
 Description
 ***********
diff --git a/docs/man/commands/search.rst b/docs/man/commands/search.rst
index 00eebc4..b3c5298 100644
--- a/docs/man/commands/search.rst
+++ b/docs/man/commands/search.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-==========
-pip-search
-==========
+===========
+pip3-search
+===========
 
 Description
 ***********
diff --git a/docs/man/commands/show.rst b/docs/man/commands/show.rst
index ce87ef9..57f81c1 100644
--- a/docs/man/commands/show.rst
+++ b/docs/man/commands/show.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-========
-pip-show
-========
+=========
+pip3-show
+=========
 
 Description
 ***********
diff --git a/docs/man/commands/uninstall.rst b/docs/man/commands/uninstall.rst
index 734ce56..c59fca1 100644
--- a/docs/man/commands/uninstall.rst
+++ b/docs/man/commands/uninstall.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-=============
-pip-uninstall
-=============
+==============
+pip3-uninstall
+==============
 
 Description
 ***********
diff --git a/docs/man/commands/wheel.rst b/docs/man/commands/wheel.rst
index 20c3a65..e4eb02b 100644
--- a/docs/man/commands/wheel.rst
+++ b/docs/man/commands/wheel.rst
@@ -1,8 +1,8 @@
 :orphan:
 
-=========
-pip-wheel
-=========
+==========
+pip3-wheel
+==========
 
 Description
 ***********
diff --git a/docs/man/index.rst b/docs/man/index.rst
index f45f0d4..134ff3c 100644
--- a/docs/man/index.rst
+++ b/docs/man/index.rst
@@ -7,12 +7,12 @@ Man Page
 SYNOPSIS
 ********
 
-pip <command> [options]
+pip3 <command> [options]
 
 DESCRIPTION
 ***********
 
-pip is the PyPA recommended package manager for Python packages
+pip3 is the PyPA recommended package manager for Python packages
 
 OPTIONS
 *******
@@ -22,38 +22,38 @@ OPTIONS
 COMMANDS
 ********
 
-pip-install(1)
+pip3-install(1)
     Install packages.
 
-pip-download(1)
+pip3-download(1)
     Download packages.
 
-pip-uninstall(1)
+pip3-uninstall(1)
     Uninstall packages.
 
-pip-freeze(1)
+pip3-freeze(1)
     Output installed packages in requirements format.
 
-pip-lock(1)
+pip3-lock(1)
     Generate a lock file for requirements and their dependencies.
 
-pip-list(1)
+pip3-list(1)
     List installed packages.
 
-pip-show(1)
+pip3-show(1)
     Show information about installed packages.
 
-pip-check(1)
+pip3-check(1)
     Verify installed packages have compatible dependencies.
 
-pip-search(1)
+pip3-search(1)
     Search PyPI for packages.
 
-pip-wheel(1)
+pip3-wheel(1)
     Build wheels from your requirements.
 
-pip-hash(1)
+pip3-hash(1)
     Compute hashes of package archives.
 
-pip-help(1)
+pip3-help(1)
     Show help for pip commands.
